features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/index.jsx

```diff
@@ -241,6 +243,53 @@ class Details extends Component {
             });
     }
 
+    getLeftMenuItemForAPIType(apiType) {
+        const { active } = this.state;
+        const { intl } = this.props;
+        switch (apiType) {
+            case 'GRAPHQL':
+                return (
+                    <React.Fragment>
+                        <LeftMenuItem
+                            text={intl.formatMessage({
+                                id: 'Apis.Details.index.schema.definition',
+                                defaultMessage: 'Schema Definition',
+                            })}
+                            handleMenuSelect={this.handleMenuSelect}
+                            active={active}
+                            Icon={<CodeIcon />}
+                        />
+                        <LeftMenuItem
+                            text='operations'
```

Comment:
Need to localize

Reply:
Fixed with 2a6fbdd3ac9ce54ca7bc94a45328a41826836a80